src/orquestra/quantum/wip/operators/_pauli_operators.py

```diff
@@ -371,8 +366,12 @@ def is_constant(self) -> bool:
 
 
 class PauliSum:
-    def __init__(self, terms: Sequence[PauliTerm] = None):
-        if terms is None:
+    def __init__(self, terms: Union[str, PauliTerm, Sequence[PauliTerm]] = None):
+        if isinstance(terms, str):
+            terms = [PauliTerm(s.strip()) for s in re.split(r"\+(?![^(]*\))", terms)]
+        elif isinstance(terms, PauliTerm):
```

Comment:
Curious what's the intention behind adding this functionality to construct a PauliSum from a singular PauliTerm?

Reply:
You're right, I removed it, we'll add it again if it turns out to be needed.